packaging/scripts/postinstall.sh

```diff
@@ -99,3 +160,8 @@ if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-decon
 		fi
 	fi
 fi
+
+# Fail installation if checkconfig command failed and the user decided to keep the old version to inform package manager about outcome
```

Comment:
Does it make sense to put this at the end. Assuming the config check failed, the above lines would still try to activate/restart the currently running service instance. 🤔